Sources/swift-parsing-benchmark/XCTestLogs.swift

```diff
@@ -50,7 +50,7 @@ private let testCaseBody = Parse {
 struct TestCaseBody: Parser {
   func parse(
     _ input: inout Substring.UTF8View
-  ) throws -> (file: Substring.UTF8View, line: Int, message: Substring.UTF8View) {
+  ) throws -> (file: Substring.UTF8View, line: Int, message: Substring) {
```

Comment:
One unintended consequence is that a `Rest` parser seems to favor the `Substring` initializer in builder blocks now...and we're not sure why yet.

Reply:
@stephencelis you'd probably need to disfavor the `buildExpression` that returns `FromSubstring`. For me it was taking precedence over 

```swift
@inlinable
public static func buildExpression<P: Parser>(_ parser: P) -> P {
  parser
}
```